app.rb

```diff
@@ -0,0 +1,88 @@
+require './book'
+require './person'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  def list_all_books
+    Book.all.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_all_people
+    Person.all.each { |person| puts "Name: #{person.name}, Age: #{person.age}, ID: #{person.id}" }
+  end
+
+  def create_teacher
+    puts 'Name: '
+    name = gets.chomp
+    puts 'Age: '
+    age = gets.chomp
+    puts 'Specialization: '
+    specialization = gets.chomp
+    Teacher.new(age, specialization, name)
+    puts 'Teacher created successfully'
+  end
+
+  def create_student
+    puts 'Name: '
+    name = gets.chomp
+    puts 'Age: '
+    age = gets.chomp
+    puts 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp
+    parent_permission = parent_permission.downcase == 'y'
+    puts 'Classroom: '
+    classroom = gets.chomp
```

Comment:
- [ ] Hi @MarcoAurelioAntonio, I got the error on the screenshot below when I tried to create a student. Please let's fix the differences in params values.

![Screenshot from 2022-11-26 04-20-13](https://user-images.githubusercontent.com/63087491/204070582-011aac4c-8108-450e-a498-f52f6c5a78c9.png)